src/OrbitGl/ThreadStateBar.cpp

```diff
@@ -161,17 +162,31 @@ std::string ThreadStateBar::GetThreadStateSliceTooltip(PrimitiveAssembler& primi
       static_cast<const ThreadStateSliceInfo*>(user_data->custom_data_);
   uint64_t begin_ns = thread_state_slice->begin_timestamp_ns();
   uint64_t end_ns = thread_state_slice->end_timestamp_ns();
-
-  return absl::StrFormat(
+  std::string tooltip = absl::StrFormat(
       "<b>%s</b><br/>"
       "<i>Thread state</i><br/>"
       "<br/>"
-      "%s"
+      "%s",
+      GetThreadStateName(thread_state_slice->thread_state()),
+      GetThreadStateDescription(thread_state_slice->thread_state()));
+  if (thread_state_slice->was_blocked_by_thread() != 0u) {
+    std::string thread_name =
+        capture_data_->GetThreadName(thread_state_slice->was_blocked_by_thread());
+    std::string process_name =
+        capture_data_->GetThreadName(thread_state_slice->was_blocked_by_process());
+    tooltip += absl::StrFormat(
+        "<br/>"
```

Comment:
I would prefer an empty line between the description of the state and the "Was Blocked By" details.